package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "@cumulus/cumulus-message-adapter-js",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "description": "Cumulus message adapter",
-  "main": "index.js",
+  "main": "./dist/index.js",
```

Comment:
I think you're probably going to need to change this to `dist/index.js`. Also, I think you're going to need to add `"files": ["dist"]"` to make sure that everything is included.

I would recommend deploying an alpha release of this package and verifying that everything you think should be included in the npm package is there, and nothing extra.

Reply:
Agreed, once the rest of the PR commentary is sorted, I'll do that and/or test vs Core.

Reply:
d8cf8a4 for now